provision.sh

```diff
@@ -4,7 +4,11 @@ set -eu
 # echo all the executed commands.
 set -x
 
+echo -e "http://dl-cdn.alpinelinux.org/alpine/v3.10/main\n" >> /etc/apk/repositories
```

Comment:
why do you do `echo -e ... \n` instead of a plain `echo`?

Reply:
it's mostly a habit, but i'm under the impression that causes the newline to be interpreted. on the other hand, i think the trailing newline is output by default, in which case it's not necessary, right?

Reply:
Yes, a plain echo will add a newline. Also just having `echo ...>>...` is easier to read.